h2olog.cc

```diff
@@ -81,6 +81,7 @@ static void show_process(pid_t pid)
     }
     size_t nread = fread(cmdline, 1, sizeof(cmdline), f);
```

Comment:
I suppose that this `fread` should be error-checked. Because `fread` return 0 if it fails. (Then `nread--` is too large number and causes crash).

Reply:
Thanks! Indeed, it should be checked. ef78dbfc4f84f6b3b75111cfc41a2074ca39a9ba